src/crimson/os/seastore/cache.h

```diff
@@ -94,30 +94,23 @@ class Cache {
 
   retired_extent_gate_t retired_extent_gate;
 
-  /// Creates empty transaction
-  TransactionRef create_transaction() {
+  /// Creates empty transaction by source
+  TransactionRef create_transaction(
+      Transaction::src_t src = Transaction::src_t::TEST) {
     LOG_PREFIX(Cache::create_transaction);
     auto ret = std::make_unique<Transaction>(
       get_dummy_ordering_handle(),
-      false,
+      src,
       last_commit
     );
     retired_extent_gate.add_token(ret->retired_gate_token);
-    DEBUGT("created", *ret);
+    DEBUGT("created source={}", *ret, src);
     return ret;
   }
 
-  /// Creates empty weak transaction
+  /// Creates empty weak transaction (test usage only)
```

Comment:
this is not only for test, mount also need weak transaction.

Reply:
After this change only cache unit test will call this interface.

Reply:
Remove the (test usage only) portion.  I interpret it to mean that weak transactions aren't used outside of tests which is not true.  Probably should use this helper in TransactionManager::mount() anyway.